lighthouse-core/config/config.js

```diff
@@ -233,6 +233,27 @@ function expandArtifacts(artifacts) {
   return artifacts;
 }
 
+function merge(value, extension) {
```

Comment:
how about s/value/base/. communicates a little better which is being mutated, and in line with your `extendConfigJSON` below